src/common/utils/gpo.h

```diff
@@ -305,4 +380,44 @@ namespace powertoys_gpo {
         return getConfiguredValue(POLICY_ALLOW_EXPERIMENTATION);
     }
 
+    inline gpo_rule_configured_t getRunPluginEnabledValue(std::string pluginID)
+    {     
+        if (pluginID == "" || pluginID == " ")
+        {
+            // this plugin id can't exist in the registry
+            return gpo_rule_configured_not_configured;
+        }
+
+        std::wstring plugin_id(pluginID.begin(), pluginID.end());
+        auto individual_plugin_setting = getPolicyListValue(POWER_LAUNCHER_INDIVIDUAL_PLUGIN_ENABLED_LIST_PATH, plugin_id);
+        
+        if (individual_plugin_setting.has_value())
+        {
+            if (*individual_plugin_setting == L"0")
+            {
+                // force disabled
+                return gpo_rule_configured_disabled;
+            }
+            else if (*individual_plugin_setting == L"1")
+            {
+                // force enabled
+                return gpo_rule_configured_enabled;
+            }
+            else if (*individual_plugin_setting == L"2")
+            {
+                // user takes control
+                return gpo_rule_configured_not_configured;
+            }
```

Comment:
@jaimecbernardo 
My problem is that these if conditions will be never true even if the registry value contains 0, 1 or 2 and we read it correctly.

It seems that cpp not compares against the text in the `std:wstring` variable. Mybe cpp compares the length or pointer.

Reply:
Hi, I've tried checking it out, but it's currently not building for me :( 
Can you please fix compile errors so that I can pick it up and debug to see where the error is coming from? ;) Thank you!

Reply:
Can you please post a screenshot?

Reply:
![image](https://github.com/microsoft/PowerToys/assets/26118718/5e93793d-82e7-4b1c-bda1-1986bfcc105e)

Reply:
I see. I think you need to `#include <optional>` in gpo.h
It's not certain that every pch  for every project will have it.

Reply:
I see what's going on. The wstring we get from registry is getting spaces past its '\0', so it's no exactly the same string, since the sizes are different. I'm adding a different retrieval method. ;)

Reply:
@htcfreek , I've pushed a commit with the fix: https://github.com/microsoft/PowerToys/pull/27468/commits/71f6257d4e2362fadc4b7f07ff485e10c113af2b

Reply:
Please let me know if that works for you :)

Reply:
Yeah. It's working. Thank you so much.